src/main/webapp/script.js

```diff
@@ -11,3 +11,81 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+var jsonPlacehold = '{ "questions" : [' +
```

Comment:
This should probably be parsed in the onLoad() function rather than globally to better match what it will be doing once this is connected to the server.